README.md

```diff
@@ -512,15 +518,56 @@ Default: `null`
 
 Type: `string`
 
-#### `mssql_ha_hacluster_password`
+#### `mssql_ha_cluster_run_role`
+
+Whether to run the `ha_cluster` role from this role.
+
+Note that the `ha_cluster` role has the following limitation:
+**The role replaces the configuration of HA Cluster on specified nodes.
+Any settings not specified in the role variables will be lost.**
+
+This means that running the `microsoft.sql.server` role re-writes existing
+Pacemaker configuration. You must verify that you want to run the `ha_cluster`
+role by setting `mssql_ha_cluster_run_role: true`.
+
+If you want to run the `ha_cluster` role independently of the
+`microsoft.sql.server` role, or run provide custom variables for the
+`ha_cluster` role, you can work around this limitation. To do this,
+run the `microsoft.sql.server` role with
+[`mssql_ha_cluster_print_vars:`](#mssql_ha_cluster_print_vars)`true` to print
+planned `ha_cluster` variables. Then you can merge the printed variables with
+your custom `ha_cluster` variables and specify the resulting set of variables
+with the `microsoft.sql.server` role invocation.
```

Comment:
What is the use case for this?  If you are using the ha_cluster role and the mssql role in the same playbook, and you are configuring mssql for ha, and you want to ensure you use the same ha parameters for both ha_cluster and mssql?  I guess this is if you first run mssql and configure it for ha, then do a subsequent playbook run with ha_cluster (with or without mssql) and you want to ensure that you do not overwrite the previous ha cluster settings used when deploying mssql?

Reply:
Or vice versa - you deploy ha_cluster role first, then you want to grab those ha_cluster variables to use with the mssql role?  If that's the case, then it seems like we need support in the ha_cluster role - have it export the settings in a format that you can use to configure something else to use ha_cluster on those same machines.  Other roles have a similar feature - the ability to export the role configuration - https://github.com/linux-system-roles/kernel_settings/pull/58 - https://github.com/linux-system-roles/firewall/pull/83 (these are wip, but I believe some other roles have already implemented this)